documentation/releaseNotes/releaseNotes.md

```diff
@@ -1,7 +1,11 @@
 Today we are releasing the next preview of the `dotnet monitor` tool. This release includes:
 
 - Load a profiler via a collection rule action (#781)
-- Get or Set an environment via a collection rule action (#1176)
+- Get or Set an environment variable via a collection rule action (#1176)
 - Add `MachineJson` option to `--output` parameter for `generatekey` command. This is useful for automating credential provisioning. (#1428)
+- Allow for simplified diagnostic port configuration (#1292)
+- Ensure `config show` displays arrays without indices (#1054)
+- ⚠️Remove netcoreapp3.1 TFM (#1267)
+- Add latest .NET 7.0 SDK (#1268)
```

Comment:
```suggestion
- Add `net7.0` TFM #1269
```

Reply:
I figured you meant for the parentheses to be there so I manually made the change; I also added the tick marks around `netcoreapp3.1` like you did for `net7.0`